corehq/util/models.py

```diff
@@ -76,15 +77,55 @@ def is_email_over_limits(cls, email):
             transient_bounce_query.count() + general_bounce_query.count() > BOUNCE_EVENT_THRESHOLD
         )
 
+    @staticmethod
+    def is_bad_email_format(email_address):
```

Comment:
Not blocking: we use `django.core.validators.validate_email` in a few places, would that work here?

Reply:
i wanted to keep it very simple to just the TLD and `@` check